Detectors/TPC/reconstruction/macro/RawClusterFinder.C

```diff
@@ -0,0 +1,550 @@
+#ifndef ALICEO2_TPC_CONVERTRAWCLUSTERS_C_
+#define ALICEO2_TPC_CONVERTRAWCLUSTERS_C_
+
+/// \file   RawClusterFinder.h
+/// \author Jens Wiechula, Jens.Wiechula@ikf.uni-frankfurt.de
+
+#include <vector>
+#include <memory>
+#include <fstream>
+
+#include "Rtypes.h"
+#include "TClonesArray.h"
+#include "TFile.h"
+#include "TSystem.h"
+#include "TGraph.h"
+#include "TLinearFitter.h"
+
+#include "TPCBase/Defs.h"
+#include "TPCBase/CalDet.h"
+#include "TPCBase/CRU.h"
+#include "TPCCalibration/CalibRawBase.h"
+#include "TPCSimulation/HwClusterer.h"
+#include "TPCSimulation/BoxClusterer.h"
+#include "TPCSimulation/ClusterContainer.h"
+#include "TPCReconstruction/DigitData.h"
+#include "TPCBase/Mapper.h"
+#include "TPCReconstruction/TrackTPC.h"
+#include "TPCSimulation/Cluster.h"
+
+#include "TCanvas.h"
+
+
+namespace o2
+{
+namespace TPC
+{
+
+
+/// \brief Raw cluster conversion
+///
+/// This class is used to produce pad wise pedestal and noise calibration data
+///
+/// origin: TPC
+/// \author Jens Wiechula, Jens.Wiechula@ikf.uni-frankfurt.de
+
+class RawClusterFinder : public CalibRawBase
+{
```

Comment:
Perhaps CalibRawBase is also not the best name. It is an interface to easily loop over raw data digit by digit, such that calibration jobs, simple cluster finders, event monitors etc. don't need to implement this over and over again. The idea was to have a few virtual functions to implement and leave the reset to the 'CalibRawBase'. Like this the actual reader can change in the background and the other code does not need modification.
We could have a short chat to see if there is a better way how to do that not using inherhitance.